### ISSUE TITLE
TFBertModel NOT learning at all! 

### ISSUE THREAD

**Issue by ivankrstev7** [huggingface/transformers#8960]
Hi, i am trying to implement a simple Keras model where the first inputs are the input_ids and the attention_mask and then i have a `TFBertModel.from_pretrained('bert-base-uncased')` layer to extract the word embeddings and everything compiles okay, but when I train the model I get a constant accuracy of 0.5 (it is a binary classification problem).
Here is how I've defined my model:
![image](https://user-images.githubusercontent.com/48191509/101362514-80b1f080-38a0-11eb-9dd3-05ab66efade6.png)
And I am using `BertTokenizer.from_pretrained('bert-base-uncased')` to prepare the dataset. I might also have a problem with how i feed the data to the model, I am not sure, so here is a scr of that too:
![image](https://user-images.githubusercontent.com/48191509/101362915-06ce3700-38a1-11eb-965a-9151228d8294.png)

**Comment by LysandreJik** [comment]
Hello, thanks for opening an issue! We try to keep the github issues for bugs/feature requests.
Could you ask your question on the [forum](https://discusss.huggingface.co) instead?

Thanks!
